product/common-src/i18n/localeDataJa.ts

```diff
@@ -98,5 +98,11 @@ export const localeDataJa: ILocaleData = {
 
   COMMON_listingConnma: '、',
 
-  defaultFileName: '名称未設定'
+  defaultFileName: '名称未設定',
+
+  RULE_animation_stamp: 'アニメーションスタンプ画像',
+  RULE_animation_main: 'アニメーションスタンプメイン画像',
+  RULE_popup: 'ポップアップスタンプ画像',
+  RULE_effect: '  エフェクトスタンプ画像',
```

Comment:
```suggestion
  RULE_effect: 'エフェクトスタンプ画像',
```
余分な空白が入ってしまっているようなので、特に意図がなければ取っておきたいですー

Reply:
@ics-nohara 
不要な空白だったの削除しました。